docs/interpreter/jdbc.md

```diff
@@ -723,6 +727,26 @@ Before Adding one of the below dependencies, check the Phoenix version first.
 </table>
 
 [Maven Repository: org.apache.tajo:tajo-jdbc](https://mvnrepository.com/artifact/org.apache.tajo/tajo-jdbc)
+## Object Interpolation
+The JDBC interpreter also supports interpolation of `ZeppelinContext` objects into the paragraph text.
+The following example shows one use of this facility:
+
+####In Scala cell:
+```
+z.put("country_code", "KR")
+    // ...
+```
+
+####In later JDBC cell:
+```
+%jdbc_interpreter_name select * from patents_list where 
+    priority_country = '{country_code}' and filing_date like '2015-__-__'
```

Comment:
is `_` in `2015-__-__` intentional?
perhaps better if the example is `2015-%`? or just pick `2015-01-01`

Reply:
I've preferred '2015-%' as using '2015-01-01' would retrieve a too small sample - and not be a realistic example.